sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Customized/Models/GetSeries.Serialization.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.IoT.TimeSeriesInsights
+{
+    /// <summary>
+    /// This class definition overrides serialization and deserialization implementation in
```

Comment:
what is it you had to change?

Reply:
I changed how TimeSeriesId is being serialized. It was being serialized as a list of object, now it's being serialized as a strongly typed object. That's what I try to explain in this comment block. Let me know if unclear and I can change it.

Reply:
Yes it is clear.